### ISSUE TITLE
psycopg2==2.6.2  - Error: could not determine PostgreSQL version from '10.0'

### ISSUE THREAD

**Issue by tony-celect-com** [psycopg/psycopg2#598]
OS: Ubuntu 14.04.5 LTS 
PostgreSQL: 9.5.9-1.pgdg14.04+1

Any one run into this issue ?

---
ii  postgresql-9.5                      9.5.9-1.pgdg14.04+1              amd64        object-relational SQL database, version 9.5 server
ii  postgresql-client-9.5               9.5.9-1.pgdg14.04+1              amd64        front-end programs for PostgreSQL 9.5
ii  postgresql-client-common            186.pgdg14.04+1                  all          manager for multiple PostgreSQL client versions
ii  postgresql-common                   186.pgdg14.04+1                  all          PostgreSQL database-cluster manager
ii  postgresql-contrib-9.5              9.5.9-1.pgdg14.04+1              amd64        additional facilities for PostgreSQL

---
Collecting psycopg2==2.6.2
/usr/local/lib/python2.7/dist-packages/pip/_vendor/requests/packages/urllib3/util/ssl_.py:318: SNIMissingWarning: An HTTPS request has been made, but the SNI (Subject Name Indication) extension to TLS is not available on this platform. This may cause the server to present an incorrect TLS certificate, which can cause validation failures. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/security.html#snimissingwarning.
  SNIMissingWarning
/usr/local/lib/python2.7/dist-packages/pip/_vendor/requests/packages/urllib3/util/ssl_.py:122: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning
  Using cached psycopg2-2.6.2.tar.gz
    Complete output from command python setup.py egg_info:
    running egg_info
    creating pip-egg-info/psycopg2.egg-info
    writing pip-egg-info/psycopg2.egg-info/PKG-INFO
    writing top-level names to pip-egg-info/psycopg2.egg-info/top_level.txt
    writing dependency_links to pip-egg-info/psycopg2.egg-info/dependency_links.txt
    writing manifest file 'pip-egg-info/psycopg2.egg-info/SOURCES.txt'
    warning: manifest_maker: standard file '-c' not found

    Error: could not determine PostgreSQL version from '10.0'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-xBnnd7/psycopg2/

**Comment by sh0ked** [comment]
Hi! I catched this problem on 2.4.5 version in TravisCI

```
Collecting psycopg2==2.4.5 (from -r requirements.txt (line 12))
  Downloading psycopg2-2.4.5.tar.gz (719kB)
    Complete output from command python setup.py egg_info:
    running egg_info
    creating pip-egg-info/psycopg2.egg-info
    writing top-level names to pip-egg-info/psycopg2.egg-info/top_level.txt
    writing dependency_links to pip-egg-info/psycopg2.egg-info/dependency_links.txt
    writing pip-egg-info/psycopg2.egg-info/PKG-INFO
    writing manifest file 'pip-egg-info/psycopg2.egg-info/SOURCES.txt'
    Error: could not determine PostgreSQL version from '10.0'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-qorDPu/psycopg2/
```

**Comment by dvarrazzo** [comment]
It is fixed in psycopg 2.7.

**Comment by npars** [comment]
If anyone was wondering the exact commit that fixed this issue: 8b96bcddff99306189152408dc8dd441020fa800

**Comment by scottcc** [comment]
Thanks very much!
